source/core/Events.cpp

```diff
@@ -548,6 +549,31 @@ static inline EventSource GetEventSourceForEventType(EventType eType) {
     return eSource;
 }
 
+void GetVIName(VirtualInstrument *vi, StringRef viName) {
```

Comment:
Since this is holding a merge back and can probably be addressed later since it would only be useful once someone else wants to use a similar helper, I'm going to add a TODO for now about moving this.